tests/main/views/test_marketplace.py

```diff
@@ -359,28 +359,24 @@ def teardown_method(self, method):
 
 class TestBriefPage(BaseBriefPageTest):
 
```

Comment:
This method was only used once so 'intrapolated' back into the test and actually perform assertions